hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/io/TestHoodieTimelineArchiver.java

```diff
@@ -1100,7 +1106,7 @@ public void testArchiveRollbacksAndCleanTestTable() throws Exception {
       testTable.doClean(cleanInstant, partitionToFileDeleteCount);
     }
 
-    for (int i = 5; i <= 13; i += 3) {
+    for (int i = 5; i <= 11; i += 2) {
```

Comment:
Can we add a comment above for why we need to jump in steps of 2?

Reply:
the jump step per run is antually the num of new commits in one loop.